frontend/src/components/ItemNavigation.tsx

```diff
@@ -59,8 +80,9 @@ function ItemNavigation(props: itemNavigationProbs) {
                            onChange={handleInputItemNumber} type="text" placeholder="item-number" name="submit"/>
                     <input value={ean}
                            onChange={handleInputEan} type="text" placeholder="ean" name="submit"/>
+                    <input value={storableValue}
```

Comment:
The input fields should not all contain the same name="submit" attribute.
Consider giving proper names and adding label-tags matching the names